DOCSP-42034-allow-custom-polymorphic-types (#78)
mongodb/docs-mongoid: source/reference/associations.txt

    has_many :items, polymorphic: true
  end

> Starting in version 9.0.2, Mongoid adds support to custom polymorphic types through

Reviewer: ```suggestion
Starting in version 9.0.2, Mongoid adds support for custom polymorphic types through
```

a global registry. You can specify alternative keys to represent different
classes, thus decoupling the code from the data. The following example specifies
the string ``"dept"`` as an alternate key for the ``Department`` class:

.. code-block:: ruby

  class Department
    include Mongoid::Document
    identify_as 'dept'
    has_many :managers, as: :unit
  end

> The ``identify_as 'dept'`` directive instructs Mongoid to store this class in the

Reviewer: S: For clarity
```suggestion
In the preceding example, the ``identify_as 'dept'`` directive instructs Mongoid to store this class in the
```

database as the string ``"dept"``. You can also specify multiple aliases. For 
example, you can specify the option as: ``identify_as 'dept', 'div', 'agency'``,
in which case the first key is the "default", and the others are used only for 
looking up records. This lets you refactor your code without breaking the associations
in your data. 

> These aliases are global. The keys you specify must be unique across your

Reviewer: S: Clarify "these"
```suggestion
Polymorphic type aliases are global. The keys you specify must be unique across your
```

entire code base. However, it is possible to register alternative resolvers, which

> must be used for different subsets of your models. In this case, the keys must

Reviewer 2: I don't think "must" is the right word here; it's less of a requirement and more of an alternative.

```suggestion
may be used for different subsets of your models. In this case, the keys must
```
